README.md

```diff
@@ -563,6 +563,7 @@ wildcards, step values or ranges:
 * `0` — Literal, matches only itself (only 0)
 * `*/15` — Step, matches any value that is a multiple (0, 15, 30, 45)
 * `0-5` — Range, matches any value within the range (0, 1, 2, 3, 4, 5)
+* `0-23/2` - Step values can be used in conjunction with ranges (0, 2, 4, 6, 8, 10, 12, 14, 16, 18, 20, 22)
```

Comment:
This is just to offer an example of the feature added with https://github.com/sorentwo/oban/pull/238